src/utils/consts.ts

```diff
@@ -1,4 +1,4 @@
-export const baseDir = "base";
+export const BASE_DIR = "base";
```

Comment:
Yes! Changed it, also 
- moved `BASE_DIR` to `copy-template-files` since it used only here.
- renamed `EXTERNAL_EXTENSION_TMP_FOLDER` to `EXTERNAL_EXTENSION_TMP_DIR` similar to other consts